TODO: glibc 2.32 now provides it. Rebuild binaries which might be
referencing the 2.21-versioned symbol, and drop this patch.

--- a/mach/Versions
+++ b/mach/Versions
@@ -57,6 +57,7 @@
   }
   GLIBC_2.21 {
     __mach_host_self_;
+    mach_print;
   }
 
   HURD_CTHREADS_0.3 {
--- a/sysdeps/mach/hurd/i386/libc.abilist
+++ b/sysdeps/mach/hurd/i386/libc.abilist
@@ -1947,6 +1947,7 @@
 GLIBC_2.2.6 xprt_unregister F
 GLIBC_2.21 __mach_host_self_ D 0x4
 GLIBC_2.21 __pthread_get_cleanup_stack F
+GLIBC_2.21 mach_print F
 GLIBC_2.21 pthread_attr_destroy F
 GLIBC_2.21 pthread_attr_getdetachstate F
 GLIBC_2.21 pthread_attr_getinheritsched F
